### PR TITLE
dist: add coverage task

### PR DESCRIPTION
**Pull request by nechaido**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -56,6 +56,7 @@
     "karma-safari-launcher": "^1.0.0",
     "karma-webpack": "^2.0.3",
     "mocha": "^3.4.2",
+    "nyc": "^11.0.2",
```
Comment (aqrln) [inline package.json]: `package-lock.json` has to be updated.

Comment (belochub) [inline package.json]: @aqrln, it already includes this package.

Comment (aqrln) [inline package.json]: @belochub oh, well then.

Comment (aqrln) [inline package.json]: @belochub nope, I checked it out. It only includes `nyc@10.3.2` while this PR requires `nyc@11`.

Comment (belochub) [inline package.json]: @aqrln, but after looking at it one more time, the major version of this package in `package-lock.json` is 10, not 11, so it should still probably be updated accordingly.

Comment (aqrln) [inline package.json]: OTOH the newer version of `tap` relies upon `nyc@11` too, so we might just update the existing dependencies.

**Comment by nechaido** [comment]
@aqrln @belochub I've added commit that updates `tap` and adds `nyc` to `package-lock.json`

**Comment by nechaido** [comment]
Landed in 015f5d9fc06652806dccffd323cd9f8fac67b315 and 3fc41dab4fa4fc96d7ee1a739727f330166042a9.

**Pull request closed without merging**